client/extensions/zoninator/state/feeds/selectors.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+const getFeedsState = ( state ) => state.extensions.zoninator.feeds;
```

Comment:
@dmsnell I think I've argued against using `get` in places like this where we can assume the subtree exists since it's created quasi-statically by a hierarchy of reducers (that are invoked during app startup) and their initial values.

IMO this is quite distinct from the 'dynamic' stuff a reducer might introduce, such as `state.extensions.zoninator.feeds[ siteId ][ zoneId ]` or whatever, where we can't know whether data are available for a given  `siteId` or `zoneId`.